e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -350,37 +350,41 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
     const CASES = {
       CountIf: ["CountIf(([Subtotal] + [Tax]) > 10)", "18,760"],
       SumIf: ["SumIf([Subtotal], ([Subtotal] + [Tax] > 20))", "1,447,850.28"],
+      SumIf2: [
+        'SumIf([Total], [Created At] > "2016-01-01") + SumIf([Subtotal], [Created At] > "2016-01-01")',
+        "2,958,809.85",
+      ],
+      CountIf2: [
+        'CountIf([Created At] > "2016-01-01") + CountIf([Created At] > "2016-01-01")',
+        "37,520",
+      ],
     };
 
     Object.entries(CASES).forEach(([filter, formula]) => {
       const [expression, result] = formula;
 
       it(`should work on custom aggregation with ${filter}`, () => {
         summarize({ mode: "notebook" });
-        // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-        cy.findByText("Custom Expression").click();
+        popover().contains("Custom Expression").click();
 
         enterCustomColumnDetails({ formula: expression });
 
         cy.findByPlaceholderText("Something nice and descriptive")
           .click()
-          .type(filter, { delay: 100 });
-
-        // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-        cy.contains(/^expected closing parenthesis/i).should("not.exist");
-        // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-        cy.contains(/^redundant input/i).should("not.exist");
+          .type(filter, { delay: 10 });
```

Comment:
[The default delay is 10](https://docs.cypress.io/api/commands/type)

```suggestion
          .type(filter);
```